fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/jbatch/bulkdata/export/system/ChunkWriter.java

```diff
@@ -209,6 +199,17 @@ public void writeItems(List<java.lang.Object> arg0) throws Exception {
     @Override
     public void open(Serializable checkpoint) throws Exception  {
         isExportPublic = FHIRConfigHelper.getBooleanProperty(FHIRConfiguration.PROPERTY_BULKDATA_BATCHJOB_ISEXPORTPUBLIC, true);
-
+        boolean isCosClientUseFhirServerTrustStore = FHIRConfigHelper
+            .getBooleanProperty(FHIRConfiguration.PROPERTY_BULKDATA_BATCHJOB_ISCOSCLIENTUSEFHIRSERVERTRUSTSTORE, false);
```

Comment:
this variable is hard to read. 

useFhirServerTrustStore and update the FHIRConfiguration